ops/aws/canary/README.md

```diff
@@ -67,10 +67,10 @@ These are the resources that had to be created/updated manually outside of CDK:
 
 ## Useful CDK commands
 Keep in mind that you might need to pass your AWS profile and the stack name in some of these commands:
-* `npm run build`   compile typescript to js
-* `npm run postinstall` deletes cdk golang templates that can result in breaking go commands due to invalid file naming pattern
-* `npm run watch`   watch for changes and compile
-* `npm run test`    perform the jest unit tests
+* `yarn run build`   compile typescript to js
+* `yarn run postinstall` deletes cdk golang templates that can result in breaking go commands due to invalid file naming pattern
+* `yarn run watch`   watch for changes and compile
+* `yarn run test`    perform the jest unit tests
```

Comment:
I don't think this CDK stuff has been changed to use yarn??

Reply:
it's a drop in replacement - it SHOULD work in most places. 

However, in this case, 'yarn' is simply a runner for the scripts in package.json - exactly the same as npm.